letterparser/parse.py

```diff
@@ -1,4 +1,38 @@
-import pypandoc
+import os
+import docker
+from letterparser.utils import unicode_encode, unicode_decode
```

Comment:
I'm not sure about the name of this module `letterparser`. Does it contain code to parse letters/characters? I'm just wondering if I'm missing some context.

Reply:
The intention of this package, which is probably not clear yet in the readme or other materials, is it will parse a `.docx` file that contains an article's decision letter and author response content and produce a JATS XML output. As a python module name, I thought a name like `decisionletterparser` was a bit long and also inaccurate because it will also parse author responses. These two parts of an article I think it is fair to call them "letters". It is not letters as in "characters", no.